biojava-structure/src/main/java/org/biojava/nbio/structure/secstruc/BasePairParameters.java

```diff
@@ -0,0 +1,403 @@
+package org.biojava.nbio.structure.secstruc;
+
+import org.biojava.nbio.structure.*;
+import org.biojava.nbio.structure.geometry.SuperPosition;
+import org.biojava.nbio.structure.geometry.SuperPositionQCP;
+import org.biojava.nbio.structure.io.PDBFileReader;
+
+import javax.vecmath.Matrix4d;
+import javax.vecmath.Point3d;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.*;
+
+import static java.lang.Math.*;
+
+/**
+ * Contributed to BioJava under its LGPL
+ * This module calculates the el Hassan-Calladine Base Pairing and Base-pair Step Parameters
+ * Citation: https://www.ncbi.nlm.nih.gov/pubmed/11601858
+ *
+ * Created by luke on 7/20/17.
+ */
+public class BasePairParameters {
+
+    private static String[] standardBases = new String[] {
```

Comment:
Standard bases are obtained from http://ndbserver.rutgers.edu/ndbmodule/archives/reports/tsukuba/Table1.html  which was also a table in the paper I cite in the header (JMB 2001 Olson et. al.).  I thought it would be better to hard code it so no issues come up with finding resources.  Previously I had them in a local folder but if someone wants to analyze helical regions it makes sense that the agreed upon standard is already in the code.